cypress/e2e/API/EngageSphere.cy.js

```diff
@@ -0,0 +1,114 @@
+/// <reference types="Cypress" />
+
+const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`;
+describe('GET /customers', () => {
+    context('Solicitações inválidas são tratadas de forma elegante', () => {
+      it('Retorna um erro e código de status 400 ao passar um valor inválido para o parâmetro page', () => {
+        cy.request({
+          method: 'GET',
+          url: `${CUSTOMERS_API_URL}?page=invalid&limit=10&size=All`,
+          failOnStatusCode: false 
+        }).then(({ status }) => {
+          expect(status).to.eq(400);
+        })
+      })
```

Comment:
Para facilitar e leitura, é recomendado deixar uma linha em branco entre um `it` e outro. Senão o código fica amontoado.

```suggestion
      })

```